classes/forbidden_shop/ForbiddenShopAPIPresenter.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+require_once __DIR__ . '/../event/LanternEvent.php';
+
+class ForbiddenShopApiPresenter {
+    /**
+     * @throws RuntimeException
+     */
+    public static function exchangeEventCurrencyResponse(ForbiddenShopManager $forbiddenShopManager, $event_name, $currency_name, $quantity)
+    {
+        return [
+            'message' => $forbiddenShopManager->exchangeEventCurrency($event_name, $currency_name, $quantity),
```

Comment:
I think this call pattern is the opposite of how it should be - Any mutations should be called from manager => presenter, not the other way around. I can fix this in the follow up tweaks